astropy/table/row.py

```diff
@@ -124,8 +124,9 @@ def as_void(self):
 
         Returns
         -------
-        void_row : np.void (unmasked) or np.ma.mvoid (masked)
-            Copy of row values
+        void_row : ``numpy.void`` or ``numpy.ma.mvoid``
```

Comment:
Did linking fail here?

Reply:
yeah. It's even worse than that. I added them to `docs/nitpicks...` file because I don't think they can be linked to at all. I would like to fix it. But... followup 🤞.